tapchannel/aux_funding_controller.go

```diff
@@ -461,6 +467,34 @@ func (p *pendingAssetFunding) addToFundingCommitment(a *asset.Asset) error {
 	return p.fundingAssetCommitment.Merge(newCommitment)
 }
 
+// addInputProofChunk adds a new proof chunk to the set of proof chunks that'll
+// be processed. If this is the last chunk for this proof, then true is
+// returned.
+func (p *pendingAssetFunding) addInputProofChunk(chunk cmsg.ProofChunk,
+) (*proof.Proof, error) {
+
+	// Collect this proof chunk with the rest of the proofs.
+	chunkID := chunk.ChunkSumID.Val
+
+	proofChunks := p.inputProofChunks[chunkID]
+	proofChunks = append(proofChunks, chunk)
+	p.inputProofChunks[chunkID] = proofChunks
```

Comment:
Can just be `p.inputProofChunks[chunkID] = append(p.inputProofChunks[chunkID], chunk)`?

Reply:
Need to reference `proofChunks` below, so ends up being the same amount of lines.